src/jit/gentree.cpp

```diff
@@ -7817,6 +7817,504 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     }
 }
 
+GenTreeOperandIterator::GenTreeOperandIterator()
+    : m_node(nullptr)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
+    , m_expandMultiRegArgs(false)
+    , m_state(-1)
+{
+}
+
```

Comment:
This is something of an open question. With tree order, `GTF_REVERSE_OPS` indicates that the defs and the uses occur in reverse order. Without tree order, we plan on using `GTF_REVERSE_OPS` to indicate only the latter condition.